testutil/ciao-down/cloudinit.go

```diff
@@ -69,6 +67,7 @@ write_files:
    path: /etc/apt/sources.list.d/docker.list
 
 runcmd:
+ - echo "127.0.0.1 singlevm" >> /etc/hosts
```

Comment:
This is dirty.  The name should be unique and configurable.  If you're adding to the file you need to remove from the file too.  It would be better to not touch the /etc/hosts file at all.